indico/modules/events/registration/templates/display/_registration_summary_blocks.html

```diff
@@ -97,6 +97,12 @@
                 <li>{{ item }}</li>
             {%- endfor %}
         </ul>
+    {% elif field.input_type == 'sessions' and friendly_data %}
```

Comment:
`_reglist.html` could also use this to not render the square brackets:

```diff
- {% elif item.id in data and data[item.id].field_data.field.input_type == 'multi_choice' %}
+ {% elif item.id in data and data[item.id].field_data.field.input_type in ('multi_choice', 'sessions') %}
```

The `render_friendly_data` macro should also have a case for the new field.

Also have a look at `RegistrantToPDF`, `RegistrantsListToPDF` and  `RegistrationFormFieldPlaceholder`